### ISSUE TITLE
args out of range error when Ement is trying load the room list

### ISSUE THREAD

**Issue by jeffbowman** [alphapapa/plz.el#27]
This is the error I get when Ement.el is connecting and trying to load the room list.  I seem to also get this error when entering a room, but not all of the messages are showing.  In other words, in a different client, I see more messages in the room than what ement is showing.   This seemed to be more related to `plz` than Ement.el which is why it is reported here. 

```
(ement--process-event): Error processing event: (args-out-of-range #s(plz-queue 5 (#s(plz-queued-request get "https://matrix-client.matrix.org/_matrix/media/r0/download/matrix.org/WEnCCdftGDmhnmoSXjMfmrtA" nil nil nil nil t binary #[257 "	\205i\0\305\306\307\310\311\312
\313&	\314\315!\203\0\316\317\320#\210\321\300!>\204+\0\322\323\324\300D\"\210\300\211\325\326\327\330#I\266\321\300!>\204F\0\322\323\324\300D\"\210\300\331\332H\236\211\203X\0\211\306\241\266\202f\0\332\331C\211\262\332HBI\266\306\262\207" [#s(ement-room "!NicAJNwJawmHrEhqZs:matrix.org" nil "t14323-3892183062_757284974_16122623_2050432868_2073053523_4156656_854594914_7184505435_0_104170" ((m\.invited_member_count . 0) (m\.joined_member_count . 263)) (#s(ement-event "$Fn5l62nc5ajdrn34U1IOCfzhK7RCVxE9p4j2nUVnScA" #s(ement-user "@alphapapa:matrix.org" nil nil nil nil nil nil nil) ((alias . "#ement.el:matrix.org")) 1607205975862 "m.room.canonical_alias" ((age . 74543846507)) "" nil nil) #s(ement-event "$4EsLqNIJ6gOcWvdlX0wV7Wsmy2r6TOzQBUM1rCjxc74" #s(ement-user "@alphapapa:matrix.org" nil nil nil nil nil nil nil) ((displayname . "alphapapa") (membership . "join")) 1607205975733 "m.room.member" ((age . 74543846636)) "@alphapapa:matrix.org" nil nil) #s(ement-event "$o71fC0rq7yjQuoqu4BJrMjcaSR3LkwZmGAAXw_4-utg" #s(ement-user "@wolfjb:matrix.org" nil nil nil nil nil nil nil) ((avatar_url . "mxc://matrix.org/aKCKoFpTzEjldIyiLLpIeBUo") (displayname . "wolfjb") (membership . "join")) 1678148191633 "m.room.member" ((age . 3601630736)) "@wolfjb:matrix.org" nil nil) #s(ement-event "$Uaj4HJDDgflMBENiBuQmmAKD8Ea1QktDHGv0owLEd70" #s(ement-user "@alphapapa:matrix.org" nil nil nil nil nil nil nil) ((creator . "@alphapapa:matrix.org") (room_version . "6")) 1607205975565 "m.room.create" ((age . 74543846804)) "" nil nil) #s(ement-event "$3nFs-sUJlQZMZYW3wXld_C54Dst4wrqFkFXM5E8INWk" #s(ement-user "@alphapapa:matrix.org" nil nil nil nil nil nil nil) ((ban . 50) (events (m\.room\.avatar . 50) (m\.room\.canonical_alias . 50) (m\.room\.encryption . 100) (m\.room\.history_visibility . 100) (m\.room\.name . 50) (m\.room\.power_levels . 100) (m\.room\.server_acl . 100) (m\.room\.tombstone . 100)) (events_default . 0) (invite . 50) (kick . 50) (redact . 50) (state_default . 50) (users (@alphapapa:matrix\.org . 100) (@treed:cybre\.space . 50)) (users_default . 0)) 1629336548041 "m.room.power_levels" ((replaces_state . "$rUPOK4rOb2SqssJYHmyKD3xRdzBHci9qgTedpngyU8Y") (age . 52413274328)) "" nil nil) #s(ement-event "$Ukrj3XgEILJH29w7Ndk_mJ7a29IWbuaHeje-oGR54bg" #s(ement-user "@nv-elisp:matrix.org" nil nil nil nil nil nil nil) ((displayname . "nv-elisp") (membership . "join")) 1649819620649 "m.room.member" ((age . 31930201720)) "@nv-elisp:matrix.org" nil nil) #s(ement-event "$4Og-WVGZqOTjylb7irTm3Xl7PJTPMJoPoXMYrnflj4k" #s(ement-user "@alphapapa:matrix.org" nil nil nil nil nil nil nil) ((name . "Ement.el")) 1607205976109 "m.room.name" ((age . 74543846260)) "" nil nil) #s(ement-event "$hxrgfaljf0iLsNB0g3Ok3hHxZr0r9p7OFJzoUfEJ_NI" #s(ement-user "@alphapapa:matrix.org" nil nil nil nil nil nil nil) ((topic . "Ement.el, a Matrix client for Emacs. | https://github.com/alphapapa/ement.el")) 1648916169886 "m.room.topic" ((replaces_state . "$l0_Mtda4d7rrj0dhrz4sE5g4MgGfXt2wrNmIvmkVRmA") (age . 32833652483)) "" nil nil) #s(ement-event "$3ySueJbmvA7tush854aLGP5sqpJn_Ks9-PBF--wNncI" #s(ement-user "@suckless_shill:matrix.org" nil nil nil nil nil nil nil) ((avatar_url . "mxc://matrix.org/cQlAomyHFjjnYsLPMbzokpuY") (displayname . "viz") (membership . "join")) 1678100431171 "m.room.member" ((age . 3649391198)) "@suckless_shill:matrix.org" nil nil) #s(ement-event "$3ie7HHy5r0-jHTmaxFXeGx4KFT8Q5kcC2HsGH-41oDk" #s(ement-user "@alphapapa:matrix.org" nil nil nil nil nil nil nil) ((join_rule . "public")) 1607205975943 "m.room.join_rules" ((age . 74543846426)) "" nil nil) #s(ement-event "$1MNlSAUk6tvBRZPH8BtLCjd6wrrpyJy-skIBRXbRGRQ" #s(ement-user "@hnb:matrix.fedibird.com" nil nil nil nil nil nil nil) ((avatar_url . "mxc://matrix.fedibird.com/gjvyaUqyUHJFXCAvtnWpwoEZ") (displayname . "ヒロ(hnb)") (membership . "join")) 1676665559630 "m.room.member" ((replaces_state . "$o937GOmoRF6-rcsCvz04lunpKcLgOZulbLtksJX3N2g")) "@hnb:matrix.fedibird.com" nil nil) #s(ement-event "$dmxmH1dR0XWlhfVwPfpJJWxk1Vs-UEPtnGdKHQgPCxs" #s(ement-user "@chops:redchops.com" nil nil nil nil nil nil nil) ((avatar_url . "mxc://redchops.com/TBpDJVFKCtzBmeRxdimKtvKo") (displayname . "chops") (membership . "join")) 1681328922850 "m.room.member" ((age . 12)) "@chops:redchops.com" nil nil) #s(ement-event "$iAdXRfGQA5bkJZCX50By0lyXEWmPo1wLiB5ndfs8iBo" #s(ement-user "@alphapapa:matrix.org" nil nil nil nil nil nil nil) ((history_visibility . "world_readable")) 1625803028276 "m.room.history_visibility" ((replaces_state . "$9ICGz7PK_jHq9jnKRfTOYGekSRuZ5YyqJDqVTNG9FKA") (age . 55946794093)) "" nil nil) #s(ement-event "$PPq6kWi5bHgNmtKtLDntHy_gAVJHeG5IeBaCzBJyrSg" #s(ement-user "@alphapapa:matrix.org" nil nil nil nil nil nil nil) ((url . "mxc://matrix.org/WEnCCdftGDmhnmoSXjMfmrtA")) 1629772782282 "m.room.avatar" ((replaces_state . "$QMFmbv_8niz-n4U1MzWyHsn-s-PgeVW3JOWsGaaglUU") (age . 51977040087)) "" nil nil)) (#s(ement-event "$Zv3zzJTA_RHXbnRU83Oodh5-i4pB56PEMiX6z144r4M" #s(ement-user "@suckless_shill:matrix.org" nil nil nil nil nil nil nil) ((body . "> <@suckless_shill:matrix.org> unfortunately, support for rich replies is still missing.  i should polish up my patch and turn it into a PR soon™

Can someone else also test https://github.com/vizs/ement.el/tree/feature/rich-replies?  The patch was done on an unclean tree of alphapapa/ement.el so I want to make sure I didn't screw up when isolating the rich reply parts from the diff. 

[ FWIW, I've been testing a slightly modified version of this patch for months now and it is stable.  ]") (format . "org.matrix.custom.html") (formatted_body . "<mx-reply><blockquote><a href=\"https://matrix.to/#/!NicAJNwJawmHrEhqZs:matrix.org/$pScUkw7sP5CF9OpnWGbAw0l5uH_bMIrYtGhD5OImBrc?via=matrix.org&via=envs.net&via=tchncs.de\">In reply to</a> <a href=\"https://matrix.to/#/@suckless_shill:matrix.org\">@suckless_shill:matrix.org</a><br>unfortunately, support for rich replies is still missing.  i should polish up my patch and turn it into a PR soon™</blockquote></mx-reply><p>Can someone else also test https://github.com/vizs/ement.el/tree/feature/rich-replies?  The patch was done on an unclean tree of alphapapa/ement.el so I want to make sure I didn't screw up when isolating the rich reply parts from the diff.</p>
<p>[ FWIW, I've been testing a slightly modified version of this patch for months now and it is stable.  ]</p>
") (m\.relates_to (m\.in_reply_to (event_id . "$pScUkw7sP5CF9OpnWGbAw0l5uH_bMIrYtGhD5OImBrc"))) (msgtype . "m.text")) 1681741449801 "m.room.message" ((age . 8372568)) nil nil nil) #s(ement-event "$_0gIvDE9QdSio0YIoAwBVtCMT5YAt-Rmkz5QBSYsgYg" #s(ement-user "@hnb:matrix.fedibird.com" nil nil nil nil nil nil nil) ((avatar_url . "mxc://matrix.fedibird.com/elCZJJrMofeWuNFujUEiJnzs") (displayname . "Hiro (hnb)") (membership . "join")) 1681694919151 "m.room.member" ((replaces_state . "$akUMaXwRcK71EdqGpH7QkMJYG5-idQ46dXqDvnpYTl0") (prev_content (avatar_url . "mxc://matrix.fedibird.com/elCZJJrMofeWuNFujUEiJnzs") (displayname . "ヒロ(hnb)") (membership . "join")) (prev_sender . "@hnb:matrix.fedibird.com")) "@hnb:matrix.fedibird.com" nil nil) #s(ement-event "$akUMaXwRcK71EdqGpH7QkMJYG5-idQ46dXqDvnpYTl0" #s(ement-user "@hnb:matrix.fedibird.com" nil nil nil nil nil nil nil) ((avatar_url . "mxc://matrix.fedibird.com/elCZJJrMofeWuNFujUEiJnzs") (displayname . "ヒロ(hnb)") (membership . "join")) 1681694902991 "m.room.member" ((replaces_state . "$1MNlSAUk6tvBRZPH8BtLCjd6wrrpyJy-skIBRXbRGRQ") (prev_content (avatar_url . "mxc://matrix.fedibird.com/gjvyaUqyUHJFXCAvtnWpwoEZ") (displayname . "ヒロ(hnb)") (membership . "join")) (prev_sender . "@hnb:matrix.fedibird.com")) "@hnb:matrix.fedibird.com" nil nil) #s(ement-event "$95QHOJhWJM3nP5NoZLbkJmtfuloVXPceVMYhOThsLbM" #s(ement-user "@alphapapa:matrix.org" nil nil nil nil nil nil nil) ((body . "That also resulted in the addition of the first test to the package, to ensure that all intended forms are recognized correctly.") (format . "org.matrix.custom.html") (formatted_body . "<p>
That also resulted in the addition of the first test to the package, to ensure that all intended forms are recognized correctly.</p>") (msgtype . "m.text")) 1681620769577 "m.room.message" ((age . 129052792)) nil nil nil) #s(ement-event "$UJ04RvtNFbqk0OOfHffZQL_fA-MUdZp--MfyfUSh2WM" #s(ement-user "@alphapapa:matrix.org" nil nil nil nil nil nil nil) ((body . "FYI, I just pushed a minor update to master that fixes the recognition of certain name forms in outgoing messages (i.e. when linkifying them).") (format . "org.matrix.custom.html") (formatted_body . "<p>
FYI, I just pushed a minor update to master that fixes the recognition of certain name forms in outgoing messages (i.e. when linkifying them).</p>") (msgtype . "m.text")) 1681620717881 "m.room.message" ((age . 129104488)) nil nil nil) #s(ement-event "$KjXov0zCwiwfwjkEPAT8aX1KfMGy-_5q9iFIoGhEqKs" #s(ement-user "@nv-elisp:matrix.org" nil nil nil nil nil nil nil) ((m\.relates_to (event_id . "$xW9xPWapIIxKE1VSUSJi-4DJJss1snTSkv6X6ymVBCw") (key . "👍") (rel_type . "m.annotation"))) 1681598358809 "m.reaction" ((age . 151463560)) nil nil nil) #s(ement-event "$xW9xPWapIIxKE1VSUSJi-4DJJss1snTSkv6X6ymVBCw" #s(ement-user "@chops:redchops.com" nil nil nil nil nil nil nil) ((body . "Yeah this time I wasn’t quite sure if it was a “me” problem or not, since this morning I also took the time to start doing a bunch of work on my init.el, mostly to give Elpaca a shot. In the process I had to shuffle a bunch of declarations around, so who knows what kind of dust shakes out of the rug at that point") (msgtype . "m.text")) 1681597793651 "m.room.message" nil nil nil nil) #s(ement-event "$ypncmZvKH3WVaZTBWP2oez3EO0tMNWAOGb5VVfuWCEs" #s(ement-user "@alphapapa:matrix.org" nil nil nil nil nil nil nil) ((body . "Well, I made yet-another bugfix release to try to avoid the byte-compilation issue.  Three bugfix releases on one package in like 15 minutes.  Not my proudest moment.") (format . "org.matrix.custom.html") (formatted_body . "<p>
Well, I made yet-another bugfix release to try to avoid the byte-compilation issue.  Three bugfix releases on one package in like 15 minutes.  Not my proudest moment.</p>") (msgtype . "m.text")) 1681597692570 "m.room.message" ((age . 152129799)) nil nil nil) #s(ement-event "$15QFXEuXVrV_kNhQq9jZAB7L4Ql-iaBAEJcnOm4AG24" #s(ement-user "@nv-elisp:matrix.org" nil nil nil nil nil nil nil) ((body . "@chops:redchops.com: If you run into any Elpaca specific trouble, feel free to open an issue at https://www.github.com/progfolio/elpaca.") (msgtype . "m.text")) 1681597606346 "m.room.message" ((age . 152216023)) nil nil nil) #s(ement-event "$Hyz0Ca0MbznzKnjFzsAq9EBrwhY0FBVnjOzeRGvjDaE" #s(ement-user "@alphapapa:matrix.org" nil nil nil nil nil nil nil) ((body . "The plz-queue struct changed, so the byte-compiler expansions won't be valid anymore.") (format . "org.matrix.custom.html") (formatted_body . "<p>
The plz-queue struct changed, so the byte-compiler expansions won't be valid anymore.</p>") (msgtype . "m.text")) 1681597447738 "m.room.message" ((age . 152374631)) nil nil nil)) nil (("m.fully_read" (type . "m.fully_read") (content (event_id . "$0qXQz9-S3hMdhPu-X_y6btYhhRTlaAD0wT9DdPVCxO0")))) ((notification_count . 0) (highlight_count . 0)) 1681741449801 nil nil nil join nil nil #s(hash-table size 65 test equal rehash-size 1.5 rehash-threshold 0.8125 data ()) ((new-events #s(ement-event "$Hyz0Ca0MbznzKnjFzsAq9EBrwhY0FBVnjOzeRGvjDaE" #s(ement-user "@alphapapa:matrix.org" nil nil nil nil nil nil nil) ((body . "The plz-queue struct changed, so the byte-compiler expansions won't be valid anymore.") (format . "org.matrix.custom.html") (formatted_body . "<p>
The plz-queue struct changed, so the byte-compiler expansions won't be valid anymore.</p>") (msgtype . "m.text")) 1681597447738 "m.room.message" ((age . 152374631)) nil nil nil) #s(ement-event "$15QFXEuXVrV_kNhQq9jZAB7L4Ql-iaBAEJcnOm4AG24" #s(ement-user "@nv-elisp:matrix.org" nil nil nil nil nil nil nil) ((body . "@chops:redchops.com: If you run into any Elpaca specific trouble, feel free to open an issue at https://www.github.com/progfolio/elpaca.") (msgtype . "m.text")) 1681597606346 "m.room.message" ((age . 152216023)) nil nil nil) #s(ement-event "$ypncmZvKH3WVaZTBWP2oez3EO0tMNWAOGb5VVfuWCEs" #s(ement-user "@alphapapa:matrix.org" nil nil nil nil nil nil nil) ((body . "Well, I made yet-another bugfix release to try to avoid the byte-compilation issue.  Three bugfix releases on one package in like 15 minutes.  Not my proudest moment.") (format . "org.matrix.custom.html") (formatted_body . "<p>
Well, I made yet-another bugfix release to try to avoid the byte-compilation issue.  Three bugfix releases on one package in like 15 minutes.  Not my proudest moment.</p>") (msgtype . "m.text")) 1681597692570 "m.room.message" ((age . 152129799)) nil nil nil) #s(ement-event "$xW9xPWapIIxKE1VSUSJi-4DJJss1snTSkv6X6ymVBCw" #s(ement-user "@chops:redchops.com" nil nil nil nil nil nil nil) ((body . "Yeah this time I wasn’t quite sure if it was a “me” problem or not, since this morning I also took the time to start doing a bunch of work on my init.el, mostly to give Elpaca a shot. In the process I had to shuffle a bunch of declarations around, so who knows what kind of dust shakes out of the rug at that point") (msgtype . "m.text")) 1681597793651 "m.room.message" nil nil nil nil) #s(ement-event "$KjXov0zCwiwfwjkEPAT8aX1KfMGy-_5q9iFIoGhEqKs" #s(ement-user "@nv-elisp:matrix.org" nil nil nil nil nil nil nil) ((m\.relates_to (event_id . "$xW9xPWapIIxKE1VSUSJi-4DJJss1snTSkv6X6ymVBCw") (key . "👍") (rel_type . "m.annotation"))) 1681598358809 "m.reaction" ((age . 151463560)) nil nil nil) #s(ement-event "$UJ04RvtNFbqk0OOfHffZQL_fA-MUdZp--MfyfUSh2WM" #s(ement-user "@alphapapa:matrix.org" nil nil nil nil nil nil nil) ((body . "FYI, I just pushed a minor update to master that fixes the recognition of certain name forms in outgoing messages (i.e. when linkifying them).") (format . "org.matrix.custom.html") (formatted_body . "<p>
FYI, I just pushed a minor update to master that fixes the recognition of certain name forms in outgoing messages (i.e. when linkifying them).</p>") (msgtype . "m.text")) 1681620717881 "m.room.message" ((age . 129104488)) nil nil nil) #s(ement-event "$95QHOJhWJM3nP5NoZLbkJmtfuloVXPceVMYhOThsLbM" #s(ement-user "@alphapapa:matrix.org" nil nil nil nil nil nil nil) ((body . "That also resulted in the addition of the first test to the package, to ensure that all intended forms are recognized correctly.") (format . "org.matrix.custom.html") (formatted_body . "<p>
That also resulted in the addition of the first test to the package, to ensure that all intended forms are recognized correctly.</p>") (msgtype . "m.text")) 1681620769577 "m.room.message" ((age . 129052792)) nil nil nil) #s(ement-event "$akUMaXwRcK71EdqGpH7QkMJYG5-idQ46dXqDvnpYTl0" #s(ement-user "@hnb:matrix.fedibird.com" nil nil nil nil nil nil nil) ((avatar_url . "mxc://matrix.fedibird.com/elCZJJrMofeWuNFujUEiJnzs") (displayname . "ヒロ(hnb)") (membership . "join")) 1681694902991 "m.room.member" ((replaces_state . "$1MNlSAUk6tvBRZPH8BtLCjd6wrrpyJy-skIBRXbRGRQ") (prev_content (avatar_url . "mxc://matrix.fedibird.com/gjvyaUqyUHJFXCAvtnWpwoEZ") (displayname . "ヒロ(hnb)") (membership . "join")) (prev_sender . "@hnb:matrix.fedibird.com")) "@hnb:matrix.fedibird.com" nil nil) #s(ement-event "$_0gIvDE9QdSio0YIoAwBVtCMT5YAt-Rmkz5QBSYsgYg" #s(ement-user "@hnb:matrix.fedibird.com" nil nil nil nil nil nil nil) ((avatar_url . "mxc://matrix.fedibird.com/elCZJJrMofeWuNFujUEiJnzs") (displayname . "Hiro (hnb)") (membership . "join")) 1681694919151 "m.room.member" ((replaces_state . "$akUMaXwRcK71EdqGpH7QkMJYG5-idQ46dXqDvnpYTl0") (prev_content (avatar_url . "mxc://matrix.fedibird.com/elCZJJrMofeWuNFujUEiJnzs") (displayname . "ヒロ(hnb)") (membership . "join")) (prev_sender . "@hnb:matrix.fedibird.com")) "@hnb:matrix.fedibird.com" nil nil) #s(ement-event "$Zv3zzJTA_RHXbnRU83Oodh5-i4pB56PEMiX6z144r4M" #s(ement-user "@suckless_shill:matrix.org" nil nil nil nil nil nil nil) ((body . "> <@suckless_shill:matrix.org> unfortunately, support for rich replies is still missing.  i should polish up my patch and turn it into a PR soon™

Can someone else also test https://github.com/vizs/ement.el/tree/feature/rich-replies?  The patch was done on an unclean tree of alphapapa/ement.el so I want to make sure I didn't screw up when isolating the rich reply parts from the diff. 

[ FWIW, I've been testing a slightly modified version of this patch for months now and it is stable.  ]") (format . "org.matrix.custom.html") (formatted_body . "<mx-reply><blockquote><a href=\"https://matrix.to/#/!NicAJNwJawmHrEhqZs:matrix.org/$pScUkw7sP5CF9OpnWGbAw0l5uH_bMIrYtGhD5OImBrc?via=matrix.org&via=envs.net&via=tchncs.de\">In reply to</a> <a href=\"https://matrix.to/#/@suckless_shill:matrix.org\">@suckless_shill:matrix.org</a><br>unfortunately, support for rich replies is still missing.  i should polish up my patch and turn it into a PR soon™</blockquote></mx-reply><p>Can someone else also test https://github.com/vizs/ement.el/tree/feature/rich-replies?  The patch was done on an unclean tree of alphapapa/ement.el so I want to make sure I didn't screw up when isolating the rich reply parts from the diff.</p>
<p>[ FWIW, I've been testing a slightly modified version of this patch for months now and it is stable.  ]</p>
") (m\.relates_to (m\.in_reply_to (event_id . "$pScUkw7sP5CF9OpnWGbAw0l5uH_bMIrYtGhD5OImBrc"))) (msgtype . "m.text")) 1681741449801 "m.room.message" ((age . 8372568)) nil nil nil)) (new-account-data-events #s(ement-event nil #s(ement-user nil nil nil nil nil nil nil nil) ((event_id . "$0qXQz9-S3hMdhPu-X_y6btYhhRTlaAD0wT9DdPVCxO0")) nil "m.fully_read" nil nil nil nil))) #s(hash-table size 65 test equal rehash-size 1.5 rehash-threshold 0.8125 data ()) #s(hash-table size 65 test eql rehash-size 1.5 rehash-threshold 0.8125 data ())) ement-room-avatars ement-room-avatar-max-width ement-room-avatar-max-height cl-struct-ement-room-tags create-image nil data-p :ascent center :max-width :max-height fboundp imagemagick-types image--set-property :type imagemagick type-of signal wrong-type-argument ement-room 13 propertize " " display room-list-avatar 18] 11 "

(fn DATA)"] nil nil nil nil nil nil #<process plz-request-curl>)) nil nil nil nil nil nil) 9)
```

**Comment by jeffbowman** [comment]
Closing.  The solution was to uninstall, restart Emacs and reinstall.

**Comment by alphapapa** [comment]
Thanks for reporting.  Sorry you had to deal with that.  I tried to solve that before anyone installed a version that caused that problem, but I guess you were too quick for me.  :)

**Comment by jeffbowman** [comment]
No worries. I had tried `(unload-feature 'plz)` et. al. And then byte compile ement.el before reporting because I thought I remembered you saying ement.el would need to be recompiled after the new plz was installed. I probably did something out of order, or maybe that wouldn't have worked anyway.  The uninstall and reboot Emacs worked, and although it's a little unfortunate to go through, not completely unexpected when updating packages.
